jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/ResourceCacheTest.java

```diff
@@ -112,67 +114,75 @@ public void testMultipleSources1() throws Exception
     {
         Path basePath = createUtilTestResources(workDir.getEmptyPathDir());
 
-        ResourceCollection rc = new ResourceCollection(
-            Resource.newResource(basePath.resolve("one")),
-            Resource.newResource(basePath.resolve("two")),
-            Resource.newResource(basePath.resolve("three")));
+        List<Resource> resourceList = Stream.of("one", "two", "three")
+            .map(basePath::resolve)
+            .map(Resource::newResource)
+            .toList();
 
-        List<Resource> r = rc.getResources();
-        MimeTypes mime = new MimeTypes();
+        try (Resource.Mount rcMount = Resource.mountCollection(resourceList))
```

Comment:
Since the `Resource`s could be created, there's no need for mounting.

Reply:
Greatly simplified